raft/raft_test.go

```diff
@@ -3338,10 +3627,19 @@ func newNetworkWithConfig(configFunc func(*Config), peers ...stateMachine) *netw
 			sm := newRaft(cfg)
 			npeers[id] = sm
 		case *raft:
+			learners := make(map[uint64]bool, 0)
```

Comment:
`learners := make(map[uint64]bool)`? Our static analysis tool complains about this :0

Reply:
oh, got it.